src/handle/compile.mli

```diff
@@ -21,13 +21,11 @@ val compile_file : string -> Sign.t
     [fname] and compiles [fname]. It is the main compiling function. It
     is called from the main program exclusively. *)
 
-open Console
-
-(** Pure wrappers around compilation functions. The provided functions
-    perform the same computations as the ones defined above, but restore
-    the state when they have finished. An optional library mapping or
-    state can be passed as argument to change the settings. *)
-module Pure : sig
-  val compile : ?lm:Path.t*string -> ?st:State.t -> bool -> Path.t -> Sign.t
-  val compile_file : ?lm:Path.t*string -> ?st:State.t -> string -> Sign.t
-end
+(** /!\ For testing only. Pure wrappers around compilation functions. The
+   provided functions perform the same computations as the ones defined above,
+   but restore the state when they have finished. An optional library mapping
+   or state can be passed as argument to change the settings. *)
+val pure_compile :
+  ?lm:Path.t*string -> ?st:Console.State.t -> bool -> Path.t -> Sign.t
+val pure_compile_file :
+  ?lm:Path.t*string -> ?st:Console.State.t -> string -> Sign.t
```

Comment:
Were these modifications necessary? Changing the API shouldn't be done without concertation IMHO

Reply:
Furthemore, it's quite frustrating to see one's work be undone without any notice.

Reply:
I removed Compile.Pure to avoid confusion with the Pure module.